packages/flutter/test/widgets/default_text_editing_shortcuts_test.dart

```diff
@@ -664,7 +664,77 @@ void main() {
       expect(state.lastIntent, isA<DeleteToNextWordBoundaryIntent>());
       expect((state.lastIntent! as DeleteToNextWordBoundaryIntent).forward, true);
     }, variant: TargetPlatformVariant.only(TargetPlatform.linux));
-  }, skip: kIsWeb); // [intended] specific tests target non-web.
+
+    testWidgets('update the editable text content when triggered on non-web', (WidgetTester tester) async {
+      final FocusNode focusNode = FocusNode();
+      addTearDown(focusNode.dispose);
+      final TextEditingController controller = TextEditingController(text: 'Flutter');
+      addTearDown(controller.dispose);
+
+      await tester.pumpWidget(MaterialApp(
+        home: Align(
+          alignment: Alignment.topLeft,
+          child: EditableText(
+            controller: controller,
+            autofocus: true,
+            focusNode: focusNode,
+            style: const TextStyle(fontSize: 10.0),
+            cursorColor: Colors.blue,
+            backgroundCursorColor: Colors.grey,
+          ),
+        ),
+      ));
+
+      // Verify that NumLock is unlocked.
+      expect(HardwareKeyboard.instance.lockModesEnabled.contains(KeyboardLockMode.numLock), isFalse);
+
+      await tester.enterText(find.byType(EditableText), 'Flutter');
+      expect(controller.selection.end, 7);
+
+      await sendKeyCombination(tester, const SingleActivator(LogicalKeyboardKey.numpad4));
+      // Verify the cursor moved to the left (numpad4).
+      expect(controller.selection.end, 6);
+    },
+      variant: TargetPlatformVariant.only(TargetPlatform.linux),
+      skip: kIsWeb); // [intended] Non-web test.
```

Comment:
Nit: Typically I'd see the closing parenthesis on the next line. Here and below.

Reply:
I did it on purpose because the analyzer was failing and complaining about `skip` requiring a comment.
I tried various format and that one was ok.
After seeing your comment, I had a closer look to the analyzer failure and, base on https://github.com/flutter/flutter/wiki/Tree-hygiene#skipped-tests, I figured out that the lint is about having an URL or [intended] in the comment, knowing this I was able to update the PR with a proper formatting.

Reply:
Ah cool, thank you!